packages/builders/scripts/inter-protocol/init-core.js

```diff
@@ -32,18 +32,21 @@ const installKeyGroups = {
   },
   main: {
     auctioneer: [
-      '../src/auction/auctioneer.js',
-      '../bundles/bundle-auctioneer.js',
+      '@agoric/inter-protocol/src/auction/auctioneer.js',
+      '../../inter-protocol/bundles/bundle-auctioneer.js',
```

Comment:
Ick, but I don't have a better suggestion. 😕

Reply:
... short of using `import-meta-resolve` to locate the `@agoric/inter-protocol` dirname, then join that with `/bundles/bundle-auctioneer.js`.  Consider it.

Reply:
I will in https://github.com/Agoric/agoric-sdk/pull/8153 that stacks on this

Reply:
I looked into it and it's worth doing, but is a bigger lift that's worth doing on its own and more holistically: https://github.com/Agoric/agoric-sdk/issues/8178